src/collective/solr/testing.py

```diff
@@ -194,11 +198,13 @@ def activateAndReindex(portal):
 COLLECTIVE_SOLR_FIXTURE = CollectiveSolrLayer()
 
 COLLECTIVE_SOLR_INTEGRATION_TESTING = IntegrationTesting(
-    bases=(LEGACY_COLLECTIVE_SOLR_FIXTURE,),
+    bases=(PLONE_APP_REGISTRY_FIXTURE,
```

Comment:
@tomgross it seems the PLONE_APP_REGISTRY fixture bleeds into the other tests which leads to a situation where the solr unit tests should fail but they actually pass (because of the test isolation issue). I have a local branch where I fixed the first problem, but then most of the tests fail because the solr unit tests try to lookup a registry that does not exist. The entire test architecture of collective.solr is build around the SolrConnectionConfig. If we remove that we have to refactor a lot of things...I'm not saying it's not worth the effort. It will just be hard to find someone who actually does the work...We also still have the legacy collective solr fixture which needs to be refactored as well.